_docs/customization.md

```diff
@@ -60,18 +66,32 @@ Pages can be used like the following:
     {% endfor %}
 &lt;/ul&gt;</code></pre>{% endraw %}
 
+* The list of variables is honestly the *best* thing in the entire Documentation page.  I referenced this so many times when I started with Pico.  And yet... there's only a single, solitary link to Twig's Documentation, and it's not even called out.  If a user wants to customize their Pico theme, they're going to need more information on Twig than this.
+
 Additional to Twigs extensive list of filters, functions and tags, Pico also provides some useful additional filters to make theming easier. You can parse any Markdown string to HTML using the `markdown` filter. Arrays can be sorted by one of its keys or a arbitrary deep sub-key using the `sort_by` filter (e.g. `{% raw %}{% for page in pages|sort_by("meta:nav"|split(":")) %}...{% endfor %}{% endraw %}` iterates through all pages, ordered by the `nav` meta header; please note the `"meta:nav"|split(":")` part of the example, which passes `['meta', 'nav']` to the filter describing a key path). You can return all values of a given key or key path of an array using the `map` filter (e.g. `{% raw %}{{ pages|map("title") }}{% endraw %}` returns all page titles).
 
+* These filters feel *way* to complicated to just casually mention at the end of the variables list.
+
 You can use different templates for different content files by specifying the `Template` meta header. Simply add e.g. `Template: blog-post` to a content file and Pico will use the `blog-post.twig` file in your theme folder to render the page.
 
+* I wish I had a use for this function, really, but why would I want to rewrite my entire template when I can just have conditional sections using `if`.  I think the only way I'd use it would be if someone *else* provided a theme that had multiple templates built in.  If I were building it myself I'd never elect to rewrite everything, I'd probably just use my own meta variable.
```

Comment:
Actually I didn't want to say that you should change NotePaper, I meant it completely independent of NotePaper. :wink: For an existing theme this approach is completely fine.

However, we shouldn't recommend this approach for new themes.